lib/Styles/Gtk/Separator.scss

```diff
@@ -0,0 +1,5 @@
+separator {
```

Comment:
In the old stylesheet we had separately defined separator colors that were more subtle. Toplevel border color is pretty harsh:

```scss
$separator-color: rgba(black, 0.15);
$separator-highlight-color: rgba(white, 0.8);

@if $color-scheme == "dark" {
    $separator-color: rgba(black, 0.35);
    $separator-highlight-color: rgba(white, 0.05);
}
```